processes/fulfillURLManifest.py

```diff
@@ -62,31 +62,37 @@ def update_manifest(self, metadataObject, bucketName, currKey):
 
```

Comment:
can we update this file to be snake case?

Reply:
Sure